selfdrive/car/subaru/values.py

```diff
@@ -299,6 +303,10 @@ def init_make(self, CP: car.CarParams):
       b'\xe9\xf6F0\x00',
       b'\xe9\xf5B0\x00',
       b'\xe9\xf6B0\x00',
+      b'\xf1\x82\xe9\xf6B0\x00',
+    ],
+    (Ecu.fwdRadar, 0x7d0, None): [
```

Comment:
there is no fwd radar on subaru. you are probably mistaking it for a different ECU address on another brand

Reply:
I'm sorry for the duplicates, i try to double check but somehow the search only works half the time... Yeah, I was just pulling all the carParams ECUs and it was in there...